docs/user-guide/vpcs.md

```diff
@@ -197,3 +198,50 @@ spec:
   - from: 1000
     to: 2999
 ```
+
+## Mode
+
+VPCs can operate in two modes: L2VNI and L3VNI. L2VNI is the default mode of
+operation and represents the conventional functionality. L3VNI is designed
+for switches that lack the hardware ASIC features that allow them to operate on L2VNI encapsulation.
```

Comment:
maybe simplify a bit - `that lack the hardware support for L2VNI` b/c it's not L2VNI encapsulation technically - those switches supports VXLAN encap